File: benchmark/equiv/TrailingZeroes1.java
public class TrailingZeroes1 {
  static public int trailingZeroes2(int n) {
    int sum = 0;
    while (n >= 5) {
      sum += n / 5;
      n = n / 5;
    }
    return sum;
  }

  static public int trailingZeroes3(int n) {
    int s = 0;
    while (n > 4)
      s += (n /= 5);
    return s;
  }
}

File: src/equiv/TrailingZeroes1.ml
module JB = Javalib_pack.JBasics
module QID = QualifiedIdentity
module E = Expr

let _ =
  let x = E.Var (Var.Mk (QID.QID ["x"], Type.Int)) in

  Equiv.run "TrailingZeroes1"
            [(JB.TBasic `Int)]
            (JB.TBasic `Int)
            "trailingZeroes2"
            "trailingZeroes3"
            [x]
            [E.mk_ige x (E.Int 5)]
